README.md

```diff
@@ -1,31 +1,116 @@
-[issues](https://github.com/iterative/vscode-dvc/issues) or via any of the other
-existing [support channels](https://dvc.org/support).
-
-## Quick Start
-
-- **Step 1.**
-  [Install a supported version of DVC on your system](https://dvc.org/doc/install)
-- **Step 2.**
-  [Install the DVC extension for Visual Studio Code](https://code.visualstudio.com/docs/editor/extension-gallery).
-- **Step 3.** See Walkthrough.
+Run, compare, visualize, and track machine learning experiments right in VS
+Code. This extension uses [DVC](https://dvc.org/), an open-source data
+versioning and ML experiment management tool. No additional services or
+databases are required.
+
+![Extension Overview](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/images/overview.gif)
+
+- **Experiment tracking**: Record training data, parameters, and metrics on top
+  of [Git](https://git-scm.com/). Navigate your experiments, compare their
+  results, and find the best ML models.
+- **Visualization**: Plot performance data in a customizable dashboard including
+  one or more overlaid experiments.
+- **Live tracking**: Capture and see metrics changing in real time.
+- **Reproducibility**: Make sure that anyone can recover or confirm previous
+  experiments, and run new experiments based on their results.
+- **Data Management**: Handle and version large datasets, files, and models
+  effectively right from VS Code.
+
+> **Note**: We are in public beta and welcome early adopter feedback!  
+> Feel free to reach out [via Discord] or open [issues in GitHub].
+>
+> [issues in github]: https://github.com/iterative/vscode-dvc/issues
+> [via discord]: https://discord.gg/BGCjJHvDHt
+
+### Why prefer this extension?
+
+- Enjoy the best developer experience with the first experiment tracking
+  interface for an IDE.
+- No external servers, databases, subscriptions, etc. Data stays fully under
+  your control and your existing Git hosting is used to share and collaborate.
+- Go beyond a simple metrics dashboard with complete ML experiments that include
+  metrics, code, and data. Powered by [DVC experiment
+  versioning][dvc experiments].
+- Implement [data versioning] on top of your favorite cloud storage, such as
+  Amazon S3, Azure Blob Storage, Google Cloud Storage, NFS, etc.
+
+[dvc experiments]:
+  https://dvc.org/doc/user-guide/experiment-management/experiments-overview
+[data versioning]: https://dvc.org/doc/use-cases/versioning-data-and-model-files
+
+## Quick start
+
+1. [Install DVC](https://dvc.org/doc/install) on your system.
+2. Install [this extension] in VS Code.
+3. Follow the **Get Started** page that pops up!
+
+<!-- prettier-ignore -->
+> 💡 Feel free to try our [example DVC project] first! Opening it [with Github
+> Codespaces] will include this extension automatically.
+
+![Initial Get Started page](https://raw.githubusercontent.com/iterative/vscode-dvc/main/extension/images/walkthroughs.png)
```

Comment:
I moved images into extension/images/ and they're no longer showing. Is there something special about extension/resources? Or is this some sort of GH CDN lag?

<img width="713" alt="image" src="https://user-images.githubusercontent.com/1477535/173521101-9aa6d58e-9242-4d65-842b-64840e99d426.png">

> Could it become an issue for #1876 as well?

Reply:
Dah... these are not in `main/` yet. Should be fine once it's merged (but can't visualize in https://github.com/iterative/vscode-dvc/tree/readme#readme 😢

Reply:
@jorgeorpinel please move these back you are going to break something and we have run out of time.

Reply:
The image is outdated too.

Reply:
> The image is outdated too.

I know. But I don't have a pre-release or development version atm. Just using the latest public version on a `example-get-started` Codespace. Probably not critical, we can update it by or after merging this.

Reply:
Updated.